src/dosbox.cpp

```diff
@@ -958,16 +963,25 @@ void DOSBOX_Init() {
 	Pmulti_remain->GetSection()->Add_string("parameters", when_idle, "");
 	Pmulti_remain->Set_help(
 	        "set type of device connected to com port.\n"
-	        "Can be disabled, dummy, modem, nullmodem, directserial.\n"
+	        "Can be disabled, dummy, serialmouse, modem, nullmodem, directserial.\n"
 	        "Additional parameters must be in the same line in the form of\n"
 	        "parameter:value. Parameter for all types is irq (optional).\n"
+	        "for serialmouse:\n"
+	        "   type, can be one of:\n"
+	        "      2btn  : 2 buttons, Microsoft serial mouse\n"
+	        "      3btn  : 3 buttons, Logitech serial mouse\n"
+	        "      wheel : 3 buttons + wheel serial mouse\n"
+	        "      msm   : 3 buttons, Mouse Systems Mouse\n"
+	        "      2btn+msm, 3btn+msm, wheel+msm : autoselection\n"
```

Comment:
Minor: for consistency with the majority of other descriptions, put the `:` right after the value:

```
 2btn:   2 buttons, Microsoft serial mouse
```

Reply:
OK, changed